src/components/schedule-page/table/schedule/schedule-parts/base-cell/base-cell.models.ts

```diff
@@ -0,0 +1,42 @@
+import { VerboseDate } from "../../../../../../common-models/month-info.model";
```

Comment:
Zrobiłem ten plik głównie dlatego, żeby dodać tę funkcję do tworzenia `data-cy`, ale pomyślałem, że skoro jest taki plik, to można by było przenieść do niego też interfejsy i enumeracje, żeby nie zaśmiecać główny plik `base-cell`